### ISSUE TITLE
Compatibility with DBM timers

### ISSUE THREAD

**Issue by MysticalOS** [BigWigsMods/BigWigs#1350]
With the whole fiasco of them changing CHAT_MSG_ADDON (https://github.com/DeadlyBossMods/DBM-Retail/issues/859#issuecomment-1431977369) then not hearing back for a full day now, i refactored comms to just not rely on their api giving valid sender arg (by now sending sender as one of my own args). since I had to bulldoze sync compatibility anyways I also added a protocol version arg too so in future should blizzard force me to update comms with new args globally, i can just bump protocol version and not have to completely change addon prefix. anyways here is diff

https://github.com/DeadlyBossMods/DBM-Unified/commit/83a13dc7633e8c8b644b83aa76a73efa81f0acd1

basically you need to listen to D5 instead of D4 and change args. This change was also applied to classic flavors as well since it makes sense to keep everything the same. Apologize for inconvenience but who knows when/if will revert the change and I'm simply not going to rely on that after being told it was an intended change for regular channels (addon comms just being collateral damage).

**Comment by MysticalOS** [comment]
Oh and from my changes, you can see your own comms don't have to change, i reformat them on my end to new format DBM expects in handler, I think only real issues I can think of are
Pull Timer
Break timer
Court of Stars (littlewigs)

**Comment by thekk1** [comment]
Current release 10.0.25 don't get pull timer triggered by current Method Raid Tool pull timer. This there a way to configure this manually?

**Comment by MysticalOS** [comment]
You'd have to open that with method raid tools.

Also thanks for quick response @funkydude
